lib/ansible_tower_client/job_template.rb

```diff
@@ -10,12 +10,19 @@ def initialize(api, raw_body)
       super
     end
 
-    def launch(vars = {})
+    def launch(options = {})
```

Comment:
@bzwei I suggested `options` due to the number of optional args that could be passed in here.  In the future we'll also want to pass in survey data.